examples/rust/get_started/examples/old/12-secure-channel-over-cloud-node-forwarding-initiator.rs

```diff
@@ -11,25 +11,25 @@ async fn main(mut ctx: Context) -> Result<()> {
         "Paste the forwarding address of the secure channel here.";
 
     // Initialize the TCP Transport.
-    let _tcp = TcpTransport::create(&ctx).await?;
+    let mut node = node(ctx);
+    let _tcp = node.create_tcp_transport().await?;
```

Comment:
Does this couple nodes to the std TCP transport?
We want transport to remain pluggable

Reply:
`create_tcp_transport` is some syntactic sugar on top of `TcpTransport::create(node.context())`. 
So other transports can still be started with `XxxTransport::create(node.context())`

Reply:
I added extension methods for all transports on top of `Node` so they are all now properly pluggable (any transport can be defined independently of `Node` and can be integrated with a `node.create_xxx_transport()`)